### PR TITLE
Misc. Doc Fixes

### PR DESCRIPTION
**Pull request by CatFreed**
Making minor doc fixes that I had found. 

Most notably, I updated all of the example projects' Readmes to contain the project description and then removed all the contributing and license files (it was mentioned that we only want one copy in the top level of the repo in a previous PR)

### CODE REVIEW

**Comment by CatFreed** [comment]
[Doc Fixes](https://trello.com/c/upFu5TPk/33-doc-fixes)

File: examples/kubos-csp-example/README.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+  - A CSP client
+  - A button poll
+
+The button poll thread continually polls the board’s button to see if it has been pressed. If it has, a notification is added to a dedicated queue.
+
+The CSP client thread checks for messages on the button queue. If found, it connects to the CSP server’s port and sends a message “Hello World”.
+
+The CSP server thread checks for connections on the CSP port and then blinks the green LED if any messages are received.
+
+The CSP connection configuration is set using the included config.json file.
```
Comment (jacoffey3) [inline examples/kubos-csp-example/README.md]: Where is the config file?

File: examples/kubos-csp-example/README.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+
+STM32F407 Discovery:
+  - TX - PA9
+  - RX - PA10
+
+MSP430F5529 Launchpad
+  - TX - P3.3
+  - RX - P3.4
+
+Notes:
```
Comment (jacoffey3) [inline examples/kubos-csp-example/README.md]: These seem more like steps than notes. Is there a larger step by step that they are supposed to be referencing in the SDK docs?

File: examples/kubos-i2c-example/README.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+  - It will then write a single byte command of ‘0xE3’ to the slave and attempt to read back a three byte response.
+  - After this attempt, the application will end.
+  
+If the HTU21D sensor has been defined in the project’s config.json file, the appropriate initialization calls will be made and then the application will enter a loop.
+  - In each iteration of the loop, the application will fetch and print the current temperature and humidity data from the sensor to the default UART port.
+  
+If the BNO055 sensor has been defined in the project’s config.json file, the sensor will be initialized in NDOF (Nine Degrees Of Freedom) mode and then the application will enter a loop.
+  - In each iteration of the loop, the application will fetch and print the current position data from the sensor to the default UART port.
+
+The easiest way to get started building this is with the [Kubos SDK](http://docs.kubos.co/latest/md_docs_kubos-sdk.html).
```
Comment (jacoffey3) [inline examples/kubos-i2c-example/README.md]: Link is broken

File: examples/kubos-csp-example/README.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+1.  To successfully run this project, two boards must be used. One should use this project with the included config.json file.
+    The other should use this project, but should have config.json file edited to reverse the "my_address" and "target_address" values.
+2.  Due to a current peculiarity with the debouncing logic, the button must be pressed twice in order for the 'send message' event to occur.
+
+The easiest way to get started building this is with the [Kubos SDK](http://docs.kubos.co/latest/md_docs_kubos-sdk.html).
```
Comment (jacoffey3) [inline examples/kubos-csp-example/README.md]: link is broken

File: examples/kubos-i2c-example/README.md [hunk 5]
```diff
@@ -1,5 +1,17 @@
-# KubOS I2C Example App
+# Kubos I2C Example App
 
 This is a simple application built on top of the [KubOS RT Platform](https://github.com/kubostech/KubOS-rt) demonstrating our I2C HAL.
```
Comment (jacoffey3) [inline examples/kubos-i2c-example/README.md]: This should note that you have to have a device that is connected and configured to the generic or device specific communication unless they edit the project to interface with another device.

File: examples/kubos-i2c-example/README.md [hunk 6]
```diff
@@ -1,5 +1,17 @@
...
 
-The easiest way to get started building this is with the [KubOS SDK](http://docs.kubos.co/latest/md_docs_kubos-sdk.html).
+This application gives several examples of how to interact with I2C devices in a Kubos project:
+
```
Comment (jacoffey3) [inline examples/kubos-i2c-example/README.md]: Can you change it to be more like:
Three currently supported configurations:
1 - Generic I2C device (still requires a device that adheres to the protocol below) 
2- HTU21D sensor
3- BNO055 sensor
Can be used with other devices, but they are not natively supported.

File: examples/kubos-spi-example/README.md [hunk 7]
```diff
@@ -1,5 +1,14 @@
-# KubOS SPI Example App
+# Kubos SPI Example App
 
 This is a simple application built on top of the [KubOS RT Platform](https://github.com/kubostech/KubOS-rt) demonstrating our SPI HAL.
 
-The easiest way to get started building this is with the [KubOS SDK](http://docs.kubos.co/latest/md_docs_kubos-sdk.html).
+This application gives two examples of how to interact with SPI devices in a Kubos project:
```
Comment (jacoffey3) [inline examples/kubos-spi-example/README.md]: Similar to the I2C app. Just rewording to make it a bit clearer that the device or a comparable one is required.

File: examples/kubos-csp-example/README.md (hunk 1 above)
Comment (CatFreed) [inline examples/kubos-csp-example/README.md]: It's in the same folder as this README.md file.

File: examples/kubos-csp-example/README.md (hunk 2 above)
Comment (CatFreed) [inline examples/kubos-csp-example/README.md]: You're right, that does sound like steps. Which I decided was a good idea. So now the readme's all have basic instructions on how to build and run the project.

File: examples/kubos-i2c-example/README.md (hunk 6 above)
Comment (CatFreed) [inline examples/kubos-i2c-example/README.md]: I left this blurb mostly intact under the new "project overview" section, but reworded things within the new "project requirements" section. Let me know if it still doesn't seem right.

**Comment by CatFreed** [comment]
I added a bunch more stuff to the example's readmes.
**Note** The reference doc links are currently broken. They point to "docs.kubos.co/{whatever}", rather than "docs.kubos.co/**sphinx**/{whatever}" because our official docs (which will be released soon) will be in that location.

**Pull request merged**